cmd/spire-server/cli/entry/create_test.go

```diff
@@ -421,6 +427,7 @@ StoreSvid        : true
         ],
         "hint": "",
         "admin": true,
+		"created_at": "1547583197",
```

Comment:
Some of the added lines in this file are using tabs for whitespace, whereas the rest of the file uses spaces. Could you change the tabs to spaces to be consistent?

Reply:
This  went unnoticed :see_no_evil: